web/src/app/admin/assistants/lib.ts

```diff
@@ -7,6 +7,8 @@ interface PersonaCreationRequest {
   task_prompt: string;
   document_set_ids: number[];
   num_chunks: number | null;
+  use_recent_documents: boolean;
+  num_days: number | null;
```

Comment:
nit: if possible, should be consistent across the board as opposed to `num_days: number | null` in some places and `num_days?: number` in others